src/main/java/com/github/maxopoly/MemeMana/MemeManaDAO.java

```diff
@@ -42,10 +49,65 @@ public void addManaUnit(MemeManaUnit unit, MemeManaOwner owner) {
 			addManaUnit.setDouble(2, unit.getOriginalAmount());
 			addManaUnit.setDouble(3, unit.getFillGrade());
 			addManaUnit.setDate(4, new Date(unit.getGainTime()));
-			addManaUnit.setInt(5, owner.getID());
+			addManaUnit.setLong(5, owner.getID());
 			addManaUnit.execute();
 		} catch (SQLException e) {
 			logger.log(Level.WARNING, "Problem adding mana unit", e);
 		}
 	}
+
+	public Map<Long,MemeManaPouch> getManaPouches() {
+		try (Connection connection = getConnection();
+				PreparedStatement getManaPouches = connection
+						.prepareStatement("select * from manaUnits;")) {
+			ResultSet rs = getManaPouches.executeQuery();
```

Comment:
Put the ResultSet into the try as well. It's also a resource to close and while the autoclosing of the Connection/PreparedStatement will probably get the ResultSet as well, it's cleaner in this pattern to put the ResultSet into the try block as a resource.